packages/rpc-graphql/src/schema/instruction.ts

```diff
@@ -845,6 +845,27 @@ export const instructionTypeDefs = /* GraphQL */ `
         signers: [Address]
     }
 
+    """
+    SplToken-2022: EnableConfidentialTransferNonConfidentialCredits instruction
+    """
+    type SplTokenEnableConfidentialTransferNonConfidentialCredits implements TransactionInstruction {
+        programId: Address
+        account: Account
+        multisigOwner: Account
+        owner: Account
+        signers: [Address]
+    }
+    """
+    SplToken-2022: DisableNonConfidentialTransferConfidentialCredits instruction
+    """
+    type SplTokenDisableNonConfidentialTransferConfidentialCredits implements TransactionInstruction {
```

Comment:
I know the parsed instruction type flops around the "non" keyword (nice catch!), but let's bring the schema name to parity with the other one.

```suggestion
    """
    SplToken-2022: DisableConfidentialTransferNonConfidentialCredits instruction
    """
    type SplTokenDisableConfidentialTransferNonConfidentialCredits implements TransactionInstruction {
```